### PR TITLE
add action-misspell

### PR DESCRIPTION
**Pull request by takuan517**
Add misspell to .github/workflows/
#21

### CODE REVIEW

File: .github/misspell.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: misspell
+        uses: reviewdog/action-misspell@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          fail_on_error: true
+          locale: "US"
```
Comment (mgrachev) [inline .github/misspell.yml:16]: Please add a new line.

**Review by mgrachev — COMMENTED** [review]
@takuan517 👋 Thank you for your contribution 👍

I have left a comment. Please take a look at it 👀

**Comment by mgrachev** [comment]
@takuan517 Thank you 👍

**Comment by review-dog** [comment]
Hi, @takuan517! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintainance of reviewdog.

Thanks again!

**Pull request merged**